src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -656,7 +656,10 @@ export class List<T> implements ISpliceable<T>, IDisposable {
 	}
 
 	splice(start: number, deleteCount: number, elements: T[] = []): void {
-		this.eventBufferer.bufferEvents(() => this.spliceable.splice(start, deleteCount, elements));
+		this.eventBufferer.bufferEvents(() => {
+			this.spliceable.splice(start, deleteCount, elements);
+			this.view.domNode.setAttribute('aria-setsize', this.length.toString());
```

Comment:
Initial value is 0 and I can set it in the constructor.
Ok so I will do it in the method ouside of the buffer